api/src/main/resources/workbench-api.yaml

```diff
@@ -3293,6 +3293,21 @@ paths:
       - cohortBuilder
       description: 'Returns information on the domains of data in the workspace''s
         CDR version along with participant and concept counts'
+      operationId: findDomainInfosDepreciate
+      responses:
+        200:
+          description: information on the domains
+          schema:
+            "$ref": "#/definitions/DomainInfoDeprecateResponse"
+  "/v1/cohortbuilder/{workspaceNamespace}/{workspaceId}/criteria/domaininfo":
+    parameters:
+      - "$ref": "#/parameters/workspaceNamespace"
+      - "$ref": "#/parameters/workspaceId"
+    get:
+      tags:
+        - cohortBuilder
+      description: 'Returns information on the domains of data in the workspace''s
+          CDR version along with participant and concept counts'
       operationId: findDomainInfos
```

Comment:
I was thinking of changing the method name from findDomainInfos to findDomainCards since the new table name is domain_card. Finalized on findDomainInfos because of consistency, let me if you think otherwise

Reply:
I think it would be nicer if it was named `findDomainCards` and the url was `/v1/cohortbuilder/{workspaceNamespace}/{workspaceId}/criteria/domaincard` and it returned a DomainCard instead of DomainInfo as response object.

Reply:
If that is to much work to add to this PR we can create another ticket to followup with those changes after the UI ticket for this work is done.